assets/js/modules/adsense/components/dashboard/AdSenseDashboardWidget.js

```diff
@@ -174,7 +174,7 @@ class AdSenseDashboardWidget extends Component {
 								</div>
 							}
 							{ ! receivingData && (
-								error ? getDataErrorComponent( _x( 'AdSense', 'Service name', 'google-site-kit' ), error, true, true, true, errorObj ) : getNoDataComponent( _x( 'AdSense', 'Service name', 'google-site-kit' ), true, true, true )
+								error ? getDataErrorComponent( 'adsense', errorObj, true, true, true ) : getNoDataComponent( _x( 'AdSense', 'Service name', 'google-site-kit' ), true, true, true )
```

Comment:
Let's update all the usages of `getDataErrorComponent` to still pass the error message as second parameter, and `errorObj` as last parameter where it currently is passed. The only thing that should be changed is passing the module slug instead of the module name.

Reply:
Yep, updated.